awx_collection/plugins/modules/tower_notification.py

```diff
@@ -159,7 +158,7 @@
     api_url:
       description:
         - The HipChat API URL.
-        - This parameter has been deprecated, please use 'notification_configuration' instead.
+        - The HipChat notification type has been deprecated.
```

Comment:
I don't see why we need to keep this at all, if it's hipchat-only isn't the best solution to just delete it?

Reply:
I was thinking of this as kind of a "soft fail" but maybe it's muddying the waters a bit.  I can go ahead and remove these parameters.

Reply:
I can see the merit in leaving a tombstone, but in that case I still think `module.fail_json` would be better than `module.deprecate` that it does now.

Reply:
There's actually no way to get to these `module.` warnings outside of a testing scenario, so I'm going to just go ahead and remove everything related to HipChat in the `tower_notification` module.